libs/form-builder/src/components/CoreFieldLookup.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  WrappedCheckbox,
+  WrappedInput,
+  WrappedInputSelect,
+  WrappedRadio,
+  WrappedSelect,
+  WrappedSwitch,
+  WrappedTextArea,
+} from '@daohaus/ui';
+import { SplitColumnLayout } from './SplitRender';
+
+export const CoreFieldLookup = {
```

Comment:
this is very cool -- allows for us to extend when/if we need to add and map more inputs for the builder